BackofficeBundle/Resources/translations/messages.en.yml

```diff
@@ -94,6 +94,9 @@ open_orchestra_backoffice:
                 date: Date
                 choice: Choice
                 content: Content choice
+                swchoff:
```

Comment:
buttons are global at all form, so i think it's better to add translation in the same level than form.submit or form.edit etc